worker/src/main/scala/org/apache/celeborn/service/deploy/worker/profiler/JVMProfiler.scala

```diff
@@ -77,4 +82,24 @@ class JVMProfiler(conf: CelebornConf) extends Logging {
       })
     }
   }
+
+  private def extractionDir: Path = {
+    val extractionDirectory = applicationsDir
+    if (extractionDirectory.toFile.exists()) {
+      extractionDirectory
+    } else {
+      Utils.createTempDir(conf.workerWorkingDir, "profiler").toPath
+    }
+  }
+
+  private def applicationsDir: Path = {
```

Comment:
Copied from [AsyncProfilerLoader.java#L139-L152](https://github.com/jvm-profiling-tools/ap-loader/blob/main/src/main/java/one/profiler/AsyncProfilerLoader.java#L139-L152).

Reply:
How about adding comment in the code?